includes/class-wporg-notifications.php

```diff
@@ -341,27 +319,6 @@ public static function get_email_body( WP_Comment $comment, ?array $comment_meta
 	}
 
 
-	/**
-	 * Returns the comments in the thread, including the last one.
-	 *
-	 * @since 0.0.2
-	 *
-	 * @param int $comment_id Last comment of the thread.
-	 *
-	 * @return array All the comments in the thread.
-	 */
-	public static function get_parent_comments( int $comment_id ): array {
```

Comment:
@amieiro FYI there is no need to duplicate this here, I've updated the references to use the `GP_Notifications` implementation.